### PR TITLE
Add PHP 7.4 to CI/CD

### PR DESCRIPTION
**Pull request by svpernova09**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -62,7 +64,8 @@ jobs:
         uses: shivammathur/setup-php@master
         with:
           php-version: ${{ matrix.php-version }}
-          extension-csv: "intl, json, pdo"
+          pecl: true
+          extension-csv: "intl, json, pdo, ext-mbstring, ext-gd"
```
Comment (localheinz) [inline .github/workflows/continuous-integration.yml]:
```suggestion
          extension-csv: "gd, intl, json, mbstring, pdo"
```

Comment (svpernova09) [reply .github/workflows/continuous-integration.yml]: This was the first thing I tried, will give it another spin.

**Comment by localheinz** [comment]
@svpernova09 

You probably need to apply this:

```diff
diff --git a/.github/workflows/continuous-integration.yml b/.github/workflows/continuous-integration.yml
index 5b5ec18..00ed5ed 100644
--- a/.github/workflows/continuous-integration.yml
+++ b/.github/workflows/continuous-integration.yml
@@ -26,7 +26,7 @@ jobs:
         uses: shivammathur/setup-php@master
         with:
           php-version: ${{ matrix.php-version }}
-          extension-csv: "intl, json, pdo"
+          extension-csv: "gd, intl, json, mbstring, pdo"

       - name: "Cache dependencies installed with composer"
         uses: actions/cache@v1.0.0
@@ -100,7 +100,7 @@ jobs:
         uses: shivammathur/setup-php@master
         with:
           php-version: ${{ matrix.php-version }}
-          extension-csv: "intl, json, pdo, xdebug"
+          extension-csv: "gd, intl, json, mbstring, pdo"
           coverage: "xdebug"

       - name: "Cache dependencies installed with composer"
```

**Comment by localheinz** [comment]
@svpernova09 

Looks a lot better, now we only have a few failing tests!

**Comment by svpernova09** [comment]
I would love some help in figuring out why this test is failing, apparently, I'm just not able to wrap my head around it.

**Pull request merged**